web/src/auth/SignupPage.js

```diff
@@ -555,6 +547,7 @@ class SignupPage extends React.Component {
         initialValues={{
           application: application.name,
           organization: application.organization,
+          countryCode: application.organizationObj.countryCodes?.[0],
```

Comment:
what if `application.organizationObj.countryCodes` is empty array? Define a function to reuse code